server/middleware/workstack.js

```diff
@@ -79,12 +103,35 @@ async function allocateToTeam(req, res, next) {
             .map(([caseId, stageId]) => [`/case/${caseId}/stage/${stageId}/user`, { userUUID: selected_user }, {
                 headers: User.createHeaders(req.user)
             }])
-            .map(async options => await allocateUser(req, res, options));
+            .map(async options => await putToCaseworkService(req, res, options));
         await Promise.all(requests);
     }
     next();
 }
 
+async function moveTeam(req, res, next) {
+    const logger = getLogger(req.requestId);
+    logger.info('MOVING_CASE_TEAMS');
+
+    let { selected_cases = [], selected_team } = req.body;
+    if (typeof selected_cases === 'string') {
+        selected_cases = [selected_cases];
+    }
+
+    if (selected_cases.length > 0 && selected_team) {
+        const requests = selected_cases
+            .map(selected => selected.split(':'))
+            .map(([caseId, stageId]) => [`/case/${caseId}/stage/${stageId}/team`, { teamUUID: selected_team }, {
+                headers: User.createHeaders(req.user)
+            }])
+            .map(async options => await putToCaseworkService(req, res, options));
+        await Promise.all(requests);
```

Comment:
Just to check if this fails where is the error caught? Tried to follow where it would bubble up to but couldn't see immediately where it would be caught.

Reply:
Hmm that's interesting, yeah feel free to raise a ticket, that's a good spot!